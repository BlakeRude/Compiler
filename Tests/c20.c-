// C-S21
int x;
int y;

main()
{
    x = 753;
    y = 444;
    output(x++);
    output(x = y++);
    outnl();
}
