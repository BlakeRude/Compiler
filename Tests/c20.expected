Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: c20.tm
754 445
Bye.
